test/bdd/features/datalayer.feature

```diff
@@ -313,4 +313,16 @@ Feature: Data layer related features
     And DC waits for import to finish
     Given DC initiates the replication for last imported dataset
     And DC waits for public key request
-    And I wait for replications to finish
\ No newline at end of file
+    And I wait for replications to finish
+
+  @fourth
+  Scenario: Check that local query returns valid object
+    And I setup 4 nodes
+    And I start the nodes
+    And I use 1st node as DC
+    And DC imports "importers/xml_examples/Basic/01_Green_to_pink_shipment.xml" as GS1-EPCIS
```

Comment:
Can you update this bdd test and import two datasets with same ot_objects so that we can validate that when we call local query only newest one is returned